master/internal/proxy/proxy.go

```diff
@@ -73,7 +73,7 @@ func (p *Proxy) Receive(ctx *actor.Context) error {
 			URL:                  msg.URL,
 			LastRequested:        time.Now(),
 			ProxyTCP:             msg.ProxyTCP,
-			AllowUnauthenticated: msg.Unauthenticated,
+			AllowUnauthenticated: true,
```

Comment:
?

Reply:
I've totally missed this hack, and websocket auth turns out to be not functional, will need to fix that.